PlayerMatcher.Tests/Matchmaker/MatchmakerTests.cs

```diff
@@ -280,5 +284,39 @@ public void ConstructMatch_AssertsOnInvalidNumberOfPlayers()
             // Assert
             Assert.Throws<ArgumentOutOfRangeException>(() => matcher.ConstructMatch(1, -4));
         }
+
+        [Test]
+        public void ConstructMatch_NoDuplicatePlayers()
```

Comment:
nice. i didnt think about that one but it would be a pretty boring match if it was jsut you all by yourself.